kuma/javascript/ssr-server.js

```diff
@@ -84,6 +86,15 @@ app.post('/ssr/:componentName', (req, res) => {
     res.json(ssr(req.params.componentName, req.body));
 });
 
+// Important that this is defined *after* the request handlers have been
+// added otherwise Sentry won't automatically hook in.
+if (process.env.SENTRY_DSN) {
+    // The request handler must be the first middleware on the app
+    app.use(Sentry.Handlers.requestHandler());
```

Comment:
Sorry to make you update this again @peterbe! This has to go back up to the top, just after the `Sentry.init(options);`, since it has to be the first middleware for the `app` (before the `morgan` and `express.json` middleware).